lib/webhooks/webhooks.js

```diff
@@ -3,7 +3,51 @@
 var crypto = require('crypto');
 var _ = require('lodash');
 var scmp = require('scmp');
-var url = require('url');
+var urllib = require('url');
+
+/**
+ * Utility function to construct the URL string, since Node.js url library won't include standard port numbers
+ *
+ * @param {URL} parsedUrl - The parsed url object that Twilio requested on your server
+ * @returns {string} - URL with standard port number included
+ */
+function buildUrlWithStandardPort(parsedUrl) {
+  let url = '';
+  const port = parsedUrl.protocol === 'https:' ? ':443' : ':80';
```

Comment:
Should add a case for the gopher protocol (:
https://nodejs.org/api/url.html#url_url_port